lib/DDG/Spice/Seasons.pm

```diff
@@ -60,6 +60,10 @@ handle query_lc => sub {
         $country = lc $loc->country_code;
     }
 
+    # Get hemisphere
+    my $hemisphere = ($loc->latitude > 0) ? 'north' : 'south';
```

Comment:
Wouldn't this get the latitude of the person searching? This would work if I typed in "first day of spring" in Australia, but typing in "summer in australia" fails

![screen shot 2015-07-03 at 1 58 53 am](https://cloud.githubusercontent.com/assets/81969/8493382/28471df8-2127-11e5-9179-15863125b65c.png)

Maybe there's a Perl module that accepts a country and spits out the hemisphere its in?

Another solution would be to ignore countries for now, i.e., accept things like "first day of spring" but don't trigger on "summer in australia". This is not ideal, of course, but it might be good enough. Would still investigate if the above is possible, though.